package.json

```diff
@@ -203,18 +210,20 @@
     "@storybook/addon-info": "^3.4.2",
     "@storybook/addon-knobs": "^3.4.2",
     "@storybook/react": "^3.4.2",
+    "babel-cli": "^6.26.0",
     "babel-core": "^6.24.1",
     "babel-eslint": "^8.0.0",
     "babel-plugin-transform-async-to-generator": "^6.24.1",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-polyfill": "^6.23.0",
+    "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-0": "^6.24.1",
     "babel-register": "^6.7.2",
     "babelify": "^8.0.0",
     "beefy": "^2.1.5",
     "brfs": "^1.6.1",
-    "browserify": "^16.1.1",
+    "browserify": "^16.2.2",
```

Comment:
Can we drop the changes to non-babel packages? (We could move them to a different PR if we want to bump them independently?)

Reply:
See also: `ganache-core` and `node-sass` below

Reply:
@whymarrh - Solved as well, looks like the PR is good to merge.